draft-ietf-cbor-7049bis.md

```diff
@@ -1274,14 +1226,6 @@ item only to the application, or take some other type of action.
 
 ## Numbers {#numbers}
 
-An application or protocol that uses CBOR might restrict the
```

Comment:
This paragraph is still true, but probably misleading in this space.
Do we still have to say some of this somewhere?

Reply:
I say it as "CBOR-based protocols MAY specify that they treat arbitrary valid data as unexpected." in "Creating CBOR-Based Protocols".